src/config/session-management.md

```diff
@@ -36,3 +36,7 @@ To make use of its features, install the `elogind` package and make sure the
 
 If you're having any issues with elogind, [enable](./services/index.md) its
 service, as waiting for a D-Bus activation can lead to issues.
+
+There is an alternative D-Bus configuration which takes advantage of elogind for
+features such as seat detection. It requires installing the` dbus-elogind`,
```

Comment:
Typo: space before the backtick, not after it. :-)

Reply:
Oops, thanks!